tests/framework/cli/test_starters.py

```diff
@@ -1293,6 +1326,10 @@ def test_valid_tools_flag(self, fake_kedro_cli, tools, example_pipeline):
         tools = _convert_tool_names_to_numbers(selected_tools=tools)
         _assert_template_ok(result, tools=tools, example_pipeline=example_pipeline)
         _assert_requirements_ok(result, tools=tools, repo_name="new-kedro-project")
+        assert (
+            "To skip the interactive flow you can run `kedro new` with\nkedro new --name=<your-project-name> --tools=<your-project-tools> --example=<yes/no>"
```

Comment:
It would make for more text to read but I think it may help to give an example that the user can copy e.g. `kedro new --name=example-project --tools=none -example=no`

Reply:
Resolved offline